opt_einsum/backends/torch.py

```diff
@@ -31,6 +31,12 @@ def transpose(a, axes):
 def einsum(equation, *operands):
     """Variadic version of torch.einsum to match numpy api.
     """
+    # rename symbols to support PyTorch 0.4.1 and earlier,
+    # which allow only symbols a-z.
+    symbols = sorted(set(equation) - set(',->'))
+    rename = {s: get_symbol(i) for i, s in enumerate(symbols)}
+    equation = ''.join(rename.get(s, s) for s in equation)
```

Comment:
@jcmgray We do this for `einsum` as well I believe. Is it time to have a single expression which converts from global to local symbols?